### ISSUE TITLE
Oltre a offrire prodotti e ingredienti del territorio, spesso biologici e di qualità altissima, la P

### ISSUE THREAD

**Issue by ehack-italy** [emergenzeHack/covid19italia_segnalazioni#1232]
<pre><yamldata>
Descrizione: 'Oltre a offrire prodotti e ingredienti del territorio, spesso biologici
  e di qualità altissima, la Pizzeria si è adeguata a consegne che mettano in sicurezza
  sia i clienti che i fattorini (che sono in house).

  Gli operatori hanno mascherina, guanti e gel sanificante. Rispettano distanza di
  un metro e non salgono al portone, ma aspettano nell''androne del condominio.'
Email: info@pizzeriafrumento.com
Link: https://pizzeriafrumento.com/
Metodi_di_pagamento_accettati: contrassegno option_2 option_1 carta_prepagat contanti
Nome_dell_esercizio_commercial: Pizzeria Frumento
Telefono: 06 780 0012
</yamldata></pre>

**Comment by FrancescaZambi89** [comment]
@cristigalas questo più che un servizio sembra una pubblicità. sono in dubbio se accettare o meno questo tipo di segnalazioni. Tu cosa ne pensi?

**Comment by cristigalas** [comment]
@FrancescaZambi89 accettiamo. Abbiamo aperto questo nuovo form proprio per intercettare i singoli esercizi commerciali.
Ovviamente va cambiato titolo, e sempre nel titolo specificata località, ecc. E, importante, aggiungere posizione, non l’ha messa... Sai come fare?

**Comment by FrancescaZambi89** [comment]
> @FrancescaZambi89 accettiamo. Abbiamo aperto questo nuovo form proprio per intercettare i singoli esercizi commerciali.
> Ovviamente va cambiato titolo, e sempre nel titolo specificata località, ecc. E, importante, aggiungere posizione, non l’ha messa... Sai come fare?

Certo! perfetto! ci penso io. Grazie @cristigalas per questo e per tutti gli altri consigli

**Comment by cristigalas** [comment]
> > @FrancescaZambi89 accettiamo. Abbiamo aperto questo nuovo form proprio per intercettare i singoli esercizi commerciali.
> > Ovviamente va cambiato titolo, e sempre nel titolo specificata località, ecc. E, importante, aggiungere posizione, non l’ha messa... Sai come fare?
> 
> Certo! perfetto! ci penso io. Grazie @cristigalas per questo e per tutti gli altri consigli

Figurati! Grazie a te!
